src/chip.cs

```diff
@@ -0,0 +1,1767 @@
+		[Export ("stop:optionOverride:responseHandler:")]
+		void Stop (byte optionMask, byte optionOverride, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("stopWithOnOff:")]
+		void StopWithOnOff (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeCurrentLevelWithResponseHandler:")]
+		void ReadAttributeCurrentLevel (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("configureAttributeCurrentLevelWithMinInterval:maxInterval:change:responseHandler:")]
+		void ConfigureAttributeCurrentLevel (ushort minInterval, ushort maxInterval, byte change, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("reportAttributeCurrentLevelWithResponseHandler:")]
+		void ReportAttributeCurrentLevel (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeClusterRevisionWithResponseHandler:")]
+		void ReadAttributeClusterRevision (ChipResponseHandler responseHandler);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (ChipCluster), Name="CHIPLowPower")]
+	[DisableDefaultCtor]
+	interface ChipLowPower
+	{
+
+		[Export ("initWithDevice:endpoint:queue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (ChipDevice device, byte endpoint, DispatchQueue queue);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("sleep:")]
+		void Sleep (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeClusterRevisionWithResponseHandler:")]
+		void ReadAttributeClusterRevision (ChipResponseHandler responseHandler);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (ChipCluster), Name="CHIPNetworkCommissioning")]
+	[DisableDefaultCtor]
+	interface ChipNetworkCommissioning
+	{
+
+		[Export ("initWithDevice:endpoint:queue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (ChipDevice device, byte endpoint, DispatchQueue queue);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("addThreadNetwork:breadcrumb:timeoutMs:responseHandler:")]
+		void AddThreadNetwork (NSData operationalDataset, ulong breadcrumb, uint timeoutMs, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("addWiFiNetwork:credentials:breadcrumb:timeoutMs:responseHandler:")]
+		void AddWiFiNetwork (NSData ssid, NSData credentials, ulong breadcrumb, uint timeoutMs, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("disableNetwork:breadcrumb:timeoutMs:responseHandler:")]
+		void DisableNetwork (NSData networkID, ulong breadcrumb, uint timeoutMs, ChipResponseHandler responseHandler);
```

Comment:
There are other `ID` that needs to be `Id`

```suggestion
		void DisableNetwork (NSData networkId, ulong breadcrumb, uint timeoutMs, ChipResponseHandler responseHandler);
```